### PR TITLE
Assignment cmalik

### PR DESCRIPTION
**Pull request by cmalik3595**
This code is submitted as Assignment 1 of BDA602 course in SDSU.

### CODE REVIEW

**Review by Anujagagangras — CHANGES_REQUESTED** [review]
Please add libraries to requirements.in and update pip-compile --output-file=requirements.txt requirements.in --upgrade

File: src/code_unittest.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            print("Test score below threshold")
+        else:
+            print("Test score above threshold")
+        self.assertTrue(score > 0.9)
+        return
+
+    @classmethod
+    def teardownClass(cls):
+        print("Tearing down class")
+        return
```
[the remark this answers is not in the document]
Comment (cmalik3595) [reply src/code_unittest.py:47]: Added libraries. Please review.

**Review by Anujagagangras — APPROVED** [review]
changes approved.

**Comment by dafrenchyman** [comment]
You shouldn't merge the pull request in until I've gone over it.

**Pull request merged**